test/expect-tests/dune_patch/dune_patch_tests.ml

```diff
@@ -147,3 +159,15 @@ let%expect_test "patching a deleted file" =
   [%expect {|
     File foo.ml not found |}]
 ;;
+
+let%expect_test "Using a patch from 'diff' with a timestamp" =
+  test [ "foo.ml", "This is wrong\n" ] ("foo.patch", unified);
+  check "foo.ml";
+  [%expect.unreachable]
+[@@expect.uncaught_exn {|
+  (Dune_util__Report_error.Already_reported)
+  Trailing output
+  ---------------
+  Error: /tmp/build_b1b383_dune/dune_71088f_patch_test/foo.ml	2024-08-29
```

Comment:
We don't want to leak this temp path in the test

Reply:
Maybe this could be better tested with a cram test, then it is easier to post-process the output?

Reply:
You can post process output in expect tests as well. See `[%output]`.

Reply:
Should be cleaner now